src/main/java/com/tagtraum/perf/gcviewer/view/ModelChartImpl.java

```diff
@@ -62,6 +59,8 @@ public class ModelChartImpl extends JScrollPane implements ModelChart, ChangeLis
     private TimeOffsetPanel timeOffsetPanel;
     private int lastViewPortWidth = 0;
 
+    private boolean shiftPressed = false;
```

Comment:
it it still needed?

Reply:
Fixed, apologies, this is one of my first code reviews.